qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -111,6 +111,25 @@ def tag_name(self) -> str:
         self._check_vanished()
         return self._elem.tagName().lower()
 
+    def attr(self, name: str) -> str:
+        """Get 'name' attribute if exists"""
+        return self._js_dict.get('attributes', {}).get(name, '')
+
+    def matches_hint_filter(self, filterstr: str, attrs=['text']) -> str:
+        """Check if 'filter' attribute matches the value of any of the
+            'attrs' attributes """
+        for _attr in attrs:
+            if _attr == 'text':
+                matchstr = str(self).casefold()
+            else:
+                matchstr = self.attr(_attr).casefold()
+
+            if all(word in matchstr for word in filterstr.split()):
```

Comment:
Can you lift this split outside the loop, so it's not calculated for every attribute? Even better to split it before passing it in, although the debug hint might change a little (so I'm fine with leaving that off for now).